eng/common/scripts/SemVer.ps1

```diff
@@ -187,10 +212,7 @@ class AzureEngSemanticVersion {
 
   static [AzureEngSemanticVersion[]] SortVersions([AzureEngSemanticVersion[]] $versions)
   {
-    return ($versions | `
-            Sort-Object -Descending -Property `
-              Major, Minor, Patch, PrereleaseLabel, PrereleaseNumber, `
-              @{ Expression = { [int]$_.BuildNumber }; Descending = $true })
+    return $versions | Sort-Object -Descending
   }
 
   static [void] QuickTests()
```

Comment:
If you haven't already could you please do a quick run of this to make sure the basic tests we have still work the same?

Reply:
I did, yes. In fact, I had to to debug an issue I was having in the original iteration.